pkg/handler/handler.go

```diff
@@ -858,7 +858,18 @@ func (h handler) validateUser(userInfo broker.UserInfo, namespace string) (bool,
 		return false, http.StatusInternalServerError, fmt.Errorf("Unable to connect to the cluster")
 	}
 	// Retrieving the rules for the user in the namespace.
-	prs, err := openshiftClient.SubjectRulesReview(userInfo.Username, userInfo.Groups, userInfo.Extra, namespace)
+	s := userInfo.Scopes
+	if userInfo.Extra != nil {
+		scope, ok := userInfo.Extra["scopes.authorization.openshift.io"]
+		switch {
+		case ok && userInfo.Scopes != nil:
+			log.Infof("Unable to determine correct scope to use. Found both top level scope and scope in extras.")
+			return false, http.StatusForbidden, fmt.Errorf("unable to determine correct scope to use")
```

Comment:
@shawn-hurley how likely is this situation? Will we get into a situation where we're screwed and always getting forbidden? What would cause both information to come in?